doc/managing_environments/topics/alerting_sop.adoc

```diff
@@ -25,13 +25,38 @@ The syndesis-db pod is not running or is not serving requests.
 
 [source,bash,options="nowrap"]
 ----
-oc logs syndesis-db-1-7nwh9 > syndesis-db.log
+oc logs syndesis-db-1-7nwh9 -c postgresql > postgresql.log
 ----
 
 5) Capture a snapshot of the 'Fuse Online - Infrastructure - DB' Grafana dashboard. The metrics can be useful for identifying errors and performance bottlenecks.
 
 6) Start or restart syndesis-db.
 
+=== FuseOnlinePostgresExporterDown
+
+*Alert name*
+
+FuseOnlinePostgresExporterDown
+
+*Description*
+
+The syndesis-db-metrics container within the syndesis-db pod is not running or is not serving requests.
+
+**Process steps**
+
+1) Log in to the OpenShift cluster.
+
+2) Switch to project namespace where Fuse Online is installed.
+
+3) If the syndesis-db pod is still running, capture logs for analysis.
+
+[source,bash,options="nowrap"]
+----
+oc logs syndesis-db-1-7nwh9 -c syndesis-db-metrics > syndesis-db-metrics.log
+----
+
+4) Start or restart the syndesis-db pod.
```

Comment:
I think we need to be carefully here, but for now it should be ok, we can always refine this instructions as we gain some experience.